AMBuildScript

```diff
@@ -22,26 +22,55 @@ import sys
 def Normalize(path):
   return os.path.abspath(os.path.normpath(path))
 
+def SetArchFlags(compiler, arch, platform):
+  if compiler.like('gcc'):
+    if arch == 'x86':
+      compiler.cflags += ['-msse']
+    else:
+      compiler.cflags += ['-fPIC']
+
+    if arch == 'x64' or arch == 'x86_64':
```

Comment:
If possible, having one representation is cleaner... then you know you only have to check  for one thing everywhere, versus both 'x64' and 'x86_64'.